config/routes.rb

```diff
@@ -5,7 +5,8 @@
         resources :countries, only: [:index], controller: :countries
         resources :regions, only: [:index], controller: :regions
       end
-
+      resources :wb_extra, param: :iso, only: [:show], :controller => "wb_extra_country_data"
```

Comment:
please use symbol syntax here for consistency with the rest of the file, ie, `controller: :wb_extra_country_data`